Data/Sys/GameSettings/DLS.ini

```diff
@@ -1,7 +1,7 @@
 # DLSE64, DLSP64 - Star Wars: Rogue Squadron III: Rebel Strike: Limited Edition Bonus Disc (Demo)
 
 [Core]
-# Values set here will override the main Dolphin settings.
+# Values set here will override the main dolphin settings.
```

Comment:
Oh, i edited them in the past with older ini files before the pr about it was merged, i will address all these issues with a follow up commit.